packages/payment-proxy-client/src/App.tsx

```diff
@@ -59,6 +59,9 @@ function Copyright(props: any) {
 }
 
 function computePercentagePaid(info: PaymentChannelInfo): number {
+  if (info.Balance.RemainingFunds <= 0) {
```

Comment:
Do we need this? The only way we divide by 0 is if both `PaidSoFar` and `RemainingFunds` are `0`

Reply:
Yes this was wrong, good catch.